src/libcore/ops/mod.rs

```diff
@@ -189,6 +190,13 @@ pub use self::range::{RangeInclusive, RangeToInclusive};
 #[unstable(feature = "try_trait", issue = "42327")]
 pub use self::try::Try;
 
+#[unstable(feature = "generator_trait", issue = "0")]
+#[cfg(not(stage0))]
+pub use self::generator::State;
```

Comment:
`core::ops::State` on its own has no connection to generators, so we might want to export this with a more descriptive name or introduce a `core::generator` module, though this can certainly wait until the future RFC that stabilizes this feature.